docs/how-to/migrate/forkid-7-to-9.md

```diff
@@ -0,0 +1,286 @@
+
+    genesis="0xd619a27d32e3050f2265a3f58dd74c8998572812da4874aa052f0886d0dfaf47"
+    cast send -j --rpc-url "http://$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        --private-key 0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625 \
+        $mngr \
+        'addNewRollupType(address,address,uint64,uint8,bytes32,string)' \
+        "$(jq -r '.deployedTo' new-consensus-out.json)" \
+        "$(jq -r '.deployedTo' verifier-out.json)" \
+        9 0 "$genesis" "test!!!" > add-rollup-type-out.json
+    ```
+
+5. Get your new rollup type id:
+
+    ```sh
+    cat add-rollup-type-out.json | jq -r '.logs[0].topics[1]'
+    ```
+
+6. Update the rollup with the id:
+
+    ```sh
+    rollup="0x1Fe038B54aeBf558638CA51C91bC8cCa06609e91"
+    cast send -j --rpc-url "http://$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        --private-key 0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625 \
+        $mngr \
+        'updateRollup(address,uint32,bytes)' \
+        "$rollup" 2 0x > update-rollup-type-out.json
+    ```
+
+7. Verify the updated rollupid. Previously the 4th value was a `7` and now it should be a `9`.
+
+    ```sh
+    cast call --rpc-url "$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        "$(jq -r '.L1Config.polygonRollupManagerAddress' /tmp/fork-7-test/genesis.json)" \
+        "rollupIDToRollupData(uint32)(address,uint64,address,uint64,bytes32,uint64,uint64,uint64,uint64,uint64,uint64,uint8)" 1
+    ```
+
+8. Set up the data availability protcol again:
+
+    ```sh
+    rollup="0x1Fe038B54aeBf558638CA51C91bC8cCa06609e91"
+    dac="0x5A6896A98c4B7C7E8f16d177C719a1d856b9154c"
+    cast send -j \
+        --private-key "0x12d7de8621a77640c9241b2595ba78ce443d05e94090365ab3bb5e19df82c625" \
+        --rpc-url "$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)" \
+        "$rollup" 'setDataAvailabilityProtocol(address)' $dac > set-dac-out.json
+    ```
+
+## Node upgrade
+
+At this stage, the smart contracts are upgraded. However, we still need to start the nodes again. 
+
+!!! warning
+    - This procedure is very sensitive.
+    - Ensure the synchronizer starts first. 
+    
+We're going to revert the parameters back to the versions of the node that worked with fork 9, and specify that _ONLY_ stage 3 should run.
+
+1. Update the `params.yml` file as follows:
+
+    ```sh
+    diff --git a/params.yml b/params.yml
+    index c2dd446..cdb8338 100644
+    --- a/params.yml
+    +++ b/params.yml
+    @@ -8,7 +8,7 @@ deployment_suffix: "-001"
+    # The deployment process is divided into various stages.
+    # The `stages` parameter indicates the specific stages you wish the deployment to proceed through.
+    # By default, it will execute all the stages.
+    -stages: [1, 2, 3, 4, 5]
+    +stages: [3]
+
+    # Docker images and repositories used to spin up services.
+    zkevm_prover_image: hermeznetwork/zkevm-prover:v6.0.0
```

Comment:
Also mention that only `deploy_cdk_central_environment` should be set to `true` in `params.yml`.

```suggestion
    yq -Y --in-place '.deploy_l1 = false' params.yml
    yq -Y --in-place '.deploy_zkevm_contracts_on_l1 = false' params.yml
    yq -Y --in-place '.deploy_databases = false' params.yml
    yq -Y --in-place '.deploy_cdk_bridge_infra = false' params.yml
    yq -Y --in-place '.deploy_zkevm_permissionless_node = false' params.yml
    yq -Y --in-place '.deploy_observability = false' params.yml
```